src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java

```diff
@@ -4882,10 +4882,18 @@ public PreSubmitCallbackHandler<AsylumCase> internalUploadAddendumEvidenceAdminN
             (callbackStage, callback) -> {
                 final AsylumCase asylumCase = callback.getCaseDetails().getCaseData();
 
+                String appellant = "The appellant";
```

Comment:
Can this be refactored as a static constant?

Reply:
Declared as static constant now, thanks Colm